docs/pages/setup/reference/license.mdx

```diff
@@ -3,9 +3,13 @@ title: Enterprise License File
 description: Teleport enterprise license file configuration parameters and requirements
 ---
 
-Commercial Teleport subscriptions require a valid license. The license file can
+Commercial self-hosted Teleport subscriptions require a valid license. The license file can
 be downloaded from the [Teleport Customer
-Portal](https://dashboard.gravitational.com/web/login).
+Portal](https://dashboard.gravitational.com/web/login). When downloading the license file the licensed products for the Teleport Customer will list within the download screen. 
```

Comment:
```suggestion
Portal](https://dashboard.gravitational.com/web/login). When downloading the file, the products you are licensed to use will be listed in the browser. 
```
Made this suggestion to remove some repetition of "license" and be a bit more specific about what "download screen" means. Is "the browser" correct here?